package.json

```diff
@@ -1,52 +1,59 @@
 {
   "name": "rally-study-template",
-  "version": "0.4.0",
+  "version": "0.5.0",
   "scripts": {
-    "build": "rollup -c && npm run tailwind",
-    "build:developer": "rollup -c --config-enable-developer-mode && npm run tailwind",
-    "build:emulator": "rollup -c --config-enable-emulator-mode && npm run tailwind",
+    "build": "cp manifest.prod.json manifest.json && npm run build:glean && rollup -c && npm run tailwind",
+    "dev": "cp manifest.dev.json manifest.json && npm run build:glean && npm run tailwind && npm-run-all --parallel dev:extension dev:webext",
```

Comment:
Ah thanks, I'll test with a clean install. The UI showing up and disappearing is https://github.com/mozilla-rally/study-template/pull/267#issuecomment-1149236324 which I'm going to try to diagnose and fix as part of this PR.

Reply:
OK I think I figured this out, the build just takes a very long time because it first builds dist/background.js, then iterates over content and worker scripts, rolling each of them up.

I changed it to clean up dist/ on each run of `npm run dev`, and to watch `./dist/exampleWorkerScript.worker.js` instead of `./dist/background.js`, because the build takes so long and backround is the first while the worker script is the last.

However, this will break as soon as someone renames `./src/exampleWorkerScript.worker.ts`, which isn't really good enough. I wonder if we should have rollup write a temp file somewhere so we know when it's done initializing?

Reply:
ooof yea tricky.. can i suggest something cheeky: instead of `push`ing the scripts onto rollupConfig, you could `unshift` them instead so that `background.js` is always last :)

or alternatively just push the scripts first to an empty array, and then push background.js afterwards if you want to keep the O(n) runtime

Reply:
Huh, so rollup seems to do things in whatever order it likes, regardless of what order I push things into the `rollupConfig` array that it returns... I'll research it a bit more.

Reply:
OK rollup has an option for this: `--watch.onBundleEnd`. This seems to work as intended, the only minor annoyance is that it will keep restarting the browser if you close it, you need to kill the `npm run dev` command to end things, but that seems reasonable I think.